web/src/main/java/org/akaza/openclinica/controller/StudyParticipantController.java

```diff
@@ -481,7 +481,16 @@ public String startParticipantImportJob(String studyOid, String siteOid, HttpSer
 		            	ResponseSuccessListAllParticipantsByStudyDTO responseSuccess =  new ResponseSuccessListAllParticipantsByStudyDTO();
 		            	
 		            	ArrayList<StudyParticipantDTO> studyParticipantDTOs = getStudyParticipantDTOs(studyOid, siteOid,study);            	  		 	            
-		 	            responseSuccess.setStudyParticipants(studyParticipantDTOs);		 	          
+		 	            responseSuccess.setStudyParticipants(studyParticipantDTOs);
+		 	            responseSuccess.setSiteOID(siteOid);
+		 	            if (siteOid != null) {
+							StudyBean site = this.getStudyDAO().findByOid(siteOid);
+							responseSuccess.setSiteID(site.getId());
+							responseSuccess.setSiteName(site.getName());
+						} else {
```

Comment:
If siteOid is null, you should not set any values at all. Leave them as nulls

Reply:
code updated